install/install.go

```diff
@@ -903,7 +903,7 @@ func writeHelmManifests(manifests string, filename string, printYAML []interface
 		}
 	}
 
-	file, _ := os.OpenFile("kubearmor.yaml", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
+	file, _ := os.OpenFile("kubearmor.yaml", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0o600)
```

Comment:
I don't think this is a valid permission.

Reply:
it's a valid permission as Octal integer literals should use the 0o prefix on modules using Go 1.13 and later,It's a gofumpt change